modules/social_features/social_group/src/Plugin/Field/GroupGroupReferenceItemList.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace Drupal\social_group\Plugin\Field;
+
+use Drupal\Core\Field\EntityReferenceFieldItemList;
+use Drupal\Core\TypedData\ComputedItemListTrait;
+use Drupal\Core\TypedData\DataDefinitionInterface;
+use Drupal\Core\TypedData\TypedDataInterface;
+use Drupal\Core\DependencyInjection\DependencySerializationTrait;
+use Drupal\group\Entity\Group;
+use Drupal\group\Entity\GroupInterface;
+use Drupal\group\Entity\GroupRelationship;
+
+/**
+ * A computed property for the related groups.
+ */
+class GroupGroupReferenceItemList extends EntityReferenceFieldItemList {
```

Comment:
GroupGroup seems a bit odd as a name 😊

Reply:
I guess it comes from the patch of https://www.drupal.org/project/group/issues/2718195
and already fixed